streampark-console/streampark-console-service/src/main/assembly/script/data/mysql-data.sql

```diff
@@ -122,7 +122,7 @@ insert into `t_menu` values (100071, 100033, 'link view', null, null, 'externalL
 insert into `t_menu` values (100072, 100033, 'link create', null, null, 'externalLink:create', null, '1', 1, NULL, now(), now());
 insert into `t_menu` values (100073, 100033, 'link update', null, null, 'externalLink:update', null, '1', 1, NULL, now(), now());
 insert into `t_menu` values (100074, 100033, 'link delete', null, null, 'externalLink:delete', null, '1', 1, NULL, now(), now());
-
+insert into `t_menu` values (100075, 100015, 'sql delete', null, null, 'sql:delete', null, '1', 1, null, now(), now());
```

Comment:
Thanks for your contribution, we also need to put the upgrade sql into [mysql upgrade](streampark-console/streampark-console-service/src/main/assembly/script/upgrade/mysql/2.1.0.sql) and [pgsql upgrade](streampark-console/streampark-console-service/src/main/assembly/script/upgrade/pgsql/2.1.0.sql)

Reply:
Thanks for the review and sorry for missing that, updated.